server/src/main/java/org/sonarsource/ldap/server/ApacheDS.java

```diff
@@ -176,6 +186,39 @@ private ApacheDS startLdapServer() throws Exception {
     return this;
   }
 
+  private ApacheDS startKdcServer() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00112)